### PR TITLE
feat: Optimize test runner to work more smoothly

### PR DESCRIPTION
**Pull request by dploeger**
Added .might-fail feature
go dep updates
BREAKING CHANGE: Removed bitwarden feature

### CODE REVIEW

**Review by timdeluxe — APPROVED** [review]
Would have been nice, if the bitwarden removal would have been its own commit, now it hides under the test fixes.
But it is not so important...

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -107,7 +107,7 @@ jobs:
                     curl -sL "https://github.com/goss-org/goss/releases/download/${{ env.GOSSVERSION }}/goss-${GOSS_ARCH}" -o goss
                     chmod +x goss
                     go build cmd/tests/test-features.go
-                    ./test-features -f ${{ matrix.flavour }} -i test-image-${{ matrix.flavour }}-${{ matrix.arch}}:local -p ${{ matrix.arch }} -t ${PWD}/.testbed -g ${PWD}/goss -l debug
+                    ./test-features -c -f ${{ matrix.flavour }} -i test-image-${{ matrix.flavour }}-${{ matrix.arch}}:local -p ${{ matrix.arch }} -t ${PWD}/.testbed -g ${PWD}/goss -l debug
```
Comment (ThomasMinor) [inline .github/workflows/test.yml:110]: SCR: Please pick your style consistent:  I guess ${{ matrix.arch}} misses a blank.
I'd also like every variable wrote in this style ${{matrix.arch}} for the looks here (no blanks breaking the flow)
but that's your choice.

File: README.md [hunk 2]
```diff
@@ -18,7 +18,6 @@ Currently supported cloud flavours are:
 Following features and tools are supported:
 * 🐟 Fish Shell
 * 📷 AzCopy
-* 🔐 Bitwarden
```
Comment (ThomasMinor) [inline README.md:21]: You maybe should have put the removal of this feature in an own commit.

**Review by ThomasMinor — CHANGES_REQUESTED** [review]
Just one thing to optimize here

File: .github/workflows/test.yml (hunk 1 above)
Comment (dploeger) [reply .github/workflows/test.yml]: Thanks. Added the whitespace. I'd like to use the whitespace as the official Github Actions docs also use whitespaces and it also distinguishes the github actions variable expansions additionally from environment variables (e.g. `${{ matrix.arch }}` vs. `${PWD}` in this example)

File: README.md (hunk 2 above)
Comment (dploeger) [reply README.md:21]: Yeah, you're right. Tim also already mentioned that. I'll rebase the change into a single commit once @rexhinvorpsi gave his okay.

**Review by rexhinvorpsi — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dploeger) [reply README.md:21]: Done

**Pull request merged**